submissions/cynthiaNwakaeme/domManipulation/shippingCost.js

```diff
@@ -0,0 +1,48 @@
+let quantity = document.getElementById("quantityNumber"),
+    price = document.getElementById("price"),
+    shippingCost = document.getElementById("shippingCost"),
+    totalCost = document.getElementById("totalCost"),
+    button = document.getElementById("buyNow");
+
+
+
+let shippingAnalysis = (event) => {
+    if (quantity.value == "" || quantity.value <= 0) {
+        button.disabled = true;
+        button.style.backgroundColor = "rgba(218, 165, 32, 0.55)";
+        event.preventDefault();
+        return false;
+
+    } else {
+        button.disabled = false;
+        let productPrice = (quantity.value * price.innerHTML);
+
+        if (productPrice <= 1000) {
+            shippingCost.innerHTML = parseInt(100);
+            totalCost.innerHTML = parseInt(productPrice) + parseInt(shippingCost.innerHTML);
+            event.preventDefault();
+            return false;
+
+        } else if (productPrice > 1000 && productPrice <= 20000) {
+            shippingCost.innerHTML = parseInt(0.15 * productPrice);
+            totalCost.innerHTML = parseInt(productPrice) + parseInt(shippingCost.innerHTML);
+            event.preventDefault();
+            return false;
+
+        } else if (productPrice > 20000 && productPrice <= 100000) {
+            shippingCost.innerHTML = parseInt(0.20 * productPrice);
+            totalCost.innerHTML = parseInt(productPrice) + parseInt(shippingCost.innerHTML);
+            event.preventDefault();
+            return false;
+
+        } else {
+            shippingCost.innerHTML = 25000;
+            totalCost.innerHTML = parseInt(productPrice) + parseInt(shippingCost.innerHTML);
+            return true;
+        }
+    }
+};
+
+button.addEventListener("click", buttonValidate = (event) => {
```

Comment:
Well done @Cynthia-Berry 

But, you were told that `Ensure that the Shipping cost and total cost values are displayed real-time, ie, the values should be seen once the quantity information is entered WITHOUT the need to press the Buy Now button.`
, 
Here, you are listening for a `click`. Figure out what event to listen to when you type a value in a input field

Reply:
Thank You for the clarification. 
I really do appreciate.

Kindly Review my Pr.

Thanks ones more.